evento/src/views/App/index.js

```diff
@@ -13,8 +14,9 @@ class App extends Component {
 			<div className="App">
 				<Router>
 					<Switch>
-						<Route exact path='/' component={MainPage} />
+						<Route exact path='/signin' component={SignInPage} />
 						<Route exact path='/event/:eventId' component={EventPage} />
+						<Route path='/' component={MainPage} />
```

Comment:
Oh right :/ Maybe we should do two routes, one for '/' and one for '/events' with both having MainPage as the component. It would be a bit clearer since A: invalid routes would not be directed there (in your code 404 page is never fired?) and B: all routes would be explicitly defined in here so it would be easier to check all the valid routes.